mix.exs

```diff
@@ -34,6 +34,8 @@ defmodule Ecto.Mixfile do
 
   defp deps do
     [{:poolboy, "~> 1.4"},
+     {:sbroker, "~> 0.6", github: "fishcakez/sbroker", branch: "gen"},
```

Comment:
`sbroker` and `connection` should be optional. We will figure out how to make poolboy optional later (please add a TODO on top of the poolboy one).